kernel/rtlil.h

```diff
@@ -792,6 +793,7 @@ struct RTLIL::Design
 
 	int refcount_modules_;
 	dict<RTLIL::IdString, RTLIL::Module*> modules_;
+	std::vector<AST::AstNode*> packages;
```

Comment:
Added to as a member of this class since the group of packages belong to the Design context.